ee/query-service/app/server.go

```diff
@@ -291,31 +294,20 @@ func extractDashboardMetaData(path string, r *http.Request) (map[string]interfac
 		return nil, false
 	}
 
-	compositeMetricQuery, compositeMetricQueryExists := requestBody["compositeMetricQuery"]
-
-	signozMetricFound := false
+	signozMetricNotFound := false
 
-	if compositeMetricQueryExists {
-		compositeMetricQueryMap := compositeMetricQuery.(map[string]interface{})
+	signozMetricNotFound = telemetry.GetInstance().CheckSigNozMetricsV2(postData.CompositeMetricQuery)
```

Comment:
You may want to check the `json.Unmarshal` error above or `postData != nil` check here, similar to below.

Reply:
done